### ISSUE TITLE
Speed multiplier (in an ore miner) off-by-one bug?

### ISSUE THREAD

**Issue by tofof** [ValkyrieofNight/EnvironmentalTech#83]
From my own analysis using numbers sourced from [this reddit thread](https://www.reddit.com/r/feedthebeast/comments/5zc8yv/environmental_tech_void_ore_miner_modifier_figures) it seems that there might be an off-by-one error involving the case of 12 speed boosts in a void miner.

Specifically, two different test runs produced evidence that 12 speed modifiers produces the same total number of items as 11 speed modifiers + 1 accuracy modifier in a tier 4 ore miner.

One run, by the thread's OP, lasted an hour; the other (Lukas-45's) ran for 3 hours. 
<sub><sup>Note that the data table provided by Lukas-45 contains a calculation error: the first 'other items' should be 42310. (He accidentally included the number of diamond ore, 3034, in that column as well -- the raw data can be seen in the imgur album he provides.) </sub></sup>

Here are the numbers:

| | 11s, 1a | 12s | error |
:-: | -----------: | :-: | :-: |
| 1 hr | 14428 | 14433 | 0.03% |
| 3 hrs | 45344 | 45281 | 0.14% |

The key point to notice is that the total yields under each condition are within two tenths of a percent of each other in both cases. 

From the pattern of the **rest** of the data, I believe that the speed multipliers each grant a 40% _multiplicative_ improvement, i.e. the speed follows e<sup>0.336*x</sup> (where x is number of speed modifiers) ... **except that the speed of 12 modifiers is identical to 11 modifiers**. That exception makes me think off-by-one logic error, not intended effect.

Assuming my numbercrunching is accurate, the 200 tick base time should shrink to 3.5 ticks in the 12 modifier case - which compares favorably with the "minimum 4 ticks" listed in the manual. The 11 speed modifier case would shrink it to only 4.9 ticks -- **different enough that neither a rounding issue nor the minimum floor should cause identical speed in both cases**. And yet, experimentally, both are 5 ticks.

Version of ETech admittedly unknown, but I didn't spot any issues or changelog that mention changing the formula, so presumably still applies. The OP of the 8-month-old thread was using whatever version was in SF3 at the time.

TLDR picture:
![environmental tech](https://user-images.githubusercontent.com/3980843/33533204-2ac0d946-d865-11e7-947a-73516e995e3f.png)
